content/docs/for-buildpack-authors/how-to/write-buildpacks/specify-env.md

```diff
@@ -3,15 +3,108 @@ title="Specify the environment"
 weight=99
 +++
 
+Environment variables are a common way to configure various buildpacks at build-time.
```

Comment:
For the content directed at app developers, this sentence is pretty relevant. But buildpacks have other uses for them:
* to configure other buildpacks (at build-time)
* to make executables available on the path
* to configure the app (at runtime)

Reply:
added this.